src/entities/shapeEntity.js

```diff
@@ -23,9 +23,31 @@ function ShapeEntity(Interceptor,shapeObject,arguments, canvasX, canvasY) {
 
   /* return area of the shape */
   this.getObjectArea = function(objectType, arguments) {
-    var objectArea = 0;
+    let objectArea = 0;
     if (!objectType.localeCompare('arc')) {
-      objectArea = 0;
+      // area of full ellipse = PI * horizontal radius * vertical radius.
+      // therefore, area of arc = difference bet. arc's start and end radians * horizontal radius * vertical radius.
+      // the below expression is adjusted for negative values and differences in arc's start and end radians over PI*2  
+      var arcSizeInRadians = ((((arguments[5] - arguments[4]) % (PI * 2)) + (PI * 2)) % (PI * 2)); 
+      objectArea = arcSizeInRadians * arguments[2] * arguments[3] / 8;
+      if(arguments[6] === 'open' || arguments[6] === 'chord' ){
```

Comment:
any idea why it's going lower-case? I initially tried CHORD and OPEN but did not work. Is the library reading all arguments and converting to lowercase somewhere?

Reply:
did you check what it passes as in the arguments? Maybe that is lower case?